x-pack/plugins/security_solution/server/lib/telemetry/async_sender.ts

```diff
@@ -291,6 +291,14 @@ export class AsyncTelemetryEventsSender implements IAsyncTelemetryEventsSender {
         };
       }
 
+      if (event.channel === TelemetryChannel.ENDPOINT_ALERTS) {
```

Comment:
Any downsides for enriching all channels with `cluster_uuid`, `cluster_name`, and `license_info`? I think all channels benefit from it or at worst can just ignore those fields if not interested?